pkg/cmd/cli/describe/describer.go

```diff
@@ -69,6 +69,18 @@ func describerMap(c *client.Client, kclient kclientset.Interface, host string) m
 		sdnapi.Kind("EgressNetworkPolicy"):              &EgressNetworkPolicyDescriber{c},
 		authorizationapi.Kind("RoleBindingRestriction"): &RoleBindingRestrictionDescriber{c},
 	}
+
+	if withCoreGroup {
```

Comment:
@smarterclayton @deads2k not sure if this is correct, but we need this as buildapi.Kind() now returns the new API group (we want to keep legacy core group around)... problem is, that DescribableResources is used by completion generator... so without this fix, the completion will be generated twice